docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -14,3 +16,38 @@ Good examples of existing notebooks that abide by these requirements are:
 
 * `Mitigating Disparities in Ranking from Binary Data <https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb>`_
 * `Binary Classification with the UCI Credit-card Default Dataset <https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20with%20the%20UCI%20Credit-card%20Default%20Dataset.ipynb>`_
+
+Working with example notebooks
```

Comment:
I have some high-level structuring issues with this section. This is how I would structure the section:
* Maybe we can rename the section to something like: **Formatting of example notebooks**
* Start by saying that the notebooks are provided as `.py` files in the [percent format](https://jupytext.readthedocs.io/en/latest/formats.html#the-percent-format) (is there a better link to the format description?)
* Then describe what's allowed and what's not allowed in the comments
  * I am personally in favor of allowing a limited markdown in the comments (something that can be translated into ReST using simple regex), and disallowing ReST (because of .ipynb export issues)... but maybe that's exactly what the next sphinx version will accomplish?
* Then say how to easily create and edit such notebooks in Visual Studio
* Describe how to transition if you have a notebook in .ipynb (do we have a strategy here? would VS Code work? if we don't have a strategy, it's also okay to say we're not aware of a good strategy)
* Maybe also mention that if you want to continue developing an existing notebook using Jupyter, you can do so with the [Jupytext](https://jupytext.readthedocs.io/en/latest/index.html) extension (which allows you to open, run and edit `.py` files in the percent format)

Reply:
- done
- done (although I left the order as is, because the first sentence tells people why there's any kind of special formatting in the first place)
- The next sphinx-gallery version will convert ReST to markdown. I'm not sure I understand what you mean by "allowing a limited markdown" because sphinx-gallery works off of ReST and can't parse markdown (I tried).
- The strategy with jupyter is to just open the .py file. Everything will work fine. I've updated the file accordingly.
- I've found jupytext to be more trouble than it's worth. I don't use it myself because VS Code knows how to work with .py files as notebooks.